5733d9e2be6485d52ffa08870cabdee0/sandbox-ui
# MGDOBR-810: Add Cloud Provider and Cloud Region options to the create bridge dialog

JIRA: https://issues.redhat.com/browse/MGDOBR-810

This PR depends on:
- https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/899

With this PR we are adding the `Cloud Provider` and `Cloud Region` options inside the bridge creation dialog.
We only support one provider and one region, so the user can't change the default selection for both.

Steps to test this:
- run the application 
- click on the "Create Smart Events Instance" button
- the new provider and region fields are present and their default values are set
- provide a name for the instance
- click on the "Create Smart Events Instance" button inside the dialog
- the bridge is created and the payload of the request includes cloud provider and cloud region

If no regions are available (meaning the region retrieved has the `enabled` property set to `false`) the entire form is disabled and  a warning message is shown. You can test this scenario by using the mocked APIs, [manipulating mocks](https://github.com/kelvah/sandbox-ui/blob/a93d5ff3489c2cdf167cd60585b419165608bc85/mocked-api/cloudProvidersData.ts#L25) and setting `enabled=false` to the only existing region.
